docs/setting-up.md

```diff
@@ -50,15 +50,31 @@ More information can be found at [this documentation](https://help.github.com/ar
 
    # Linux/OS X
    ./install.sh --path-update true
+
    # Windows
    install.bat --path-update true
+
+   # Red Hat/CentOS/Fedora
+   sudo yum install google-cloud-sdk
+
+   # Debian/Ubuntu
+   sudo apt-get install google-cloud-sdk
    ```
    **Verification**: Run a `gcloud` command (e.g. `gcloud version`) in order to verify that you can access the SDK from the command line.
 
 1. Run this command to install App Engine Java SDK bundled with the Cloud SDK:
    ```sh
+   # Linux/OS X/Windows
    gcloud -q components install app-engine-java
+   
+   # Red Hat/CentOS/Fedora
+   sudo yum install google-cloud-sdk-app-engine-java
+   
+   # Debian/Ubuntu
+   sudo apt-get install google-cloud-sdk-app-engine-java
    ```
+   If you face any problem regarding the installation, refer to the official documentation of Google Cloud Engine for [Debian/Ubuntu](https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu) or [RHEL/CentOS/Fedora](https://cloud.google.com/sdk/docs/quickstart-redhat-centos).
```

Comment:
This is about installing the Cloud SDK; if everything was done right before, the `yum` and `apt-get` commands down here would have worked.